test/hull_test.cpp

```diff
@@ -81,15 +81,6 @@ TEST(Hull, Tictac) {
   EXPECT_NEAR(sphere.NumVert() + tictacSeg, tictac.NumVert(), 1);
 }
 
-#ifdef MANIFOLD_EXPORT
```

Comment:
I think it needs this `#ifdef` since it's using Assimp for ReadMesh.

Reply:
No, the last few `BooleanComplex` tests are wrapped inside a large `#ifdef MANIFOLD_EXPORT` block.

Reply:
Oh, got it, thanks!